apps/passport-client/src/broadcastChannel.ts

```diff
@@ -42,7 +42,7 @@ export function notifyLogoutToOtherTabs(): void {
  * set up more onmessage handlers.
  */
 export function setupBroadcastChannel(
-  dispatch: (action: Action) => Promise<void>
+  dispatch: (action: Action) => void
```

Comment:
what's the reason for making `dispatch` not async here?

Reply:
See earlier comment - the type for `Dispatcher` is non-async, and that's what gets passed in here.